src/xhydro/frequency_analysis/uncertainties.py

```diff
@@ -175,9 +175,8 @@ def calc_moments_iter(ds_samples: xr.Dataset) -> xr.Dataset:
     return xr.concat(ds_mom, dim="samples")
 
 
-def calc_q_iter(
+def _calc_q_iter_da(
     bv: str,
-    var: str,
     ds_groups: xr.Dataset,
     ds_moments_iter: xr.Dataset,
```

Comment:
Je ne peux pas le faire ici, mais il faudrait aussi changer les docstrings.
```suggestion
    da_groups: xr.DataArray,
    da_moments_iter: xr.DataArray,
```

Reply:
Je suggère ds-->da, puisqu'il s'agit maintenant de DataArrays